cluster-scope/bundles/acm-observability/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 resources:
   - ../../base/core/namespaces/open-cluster-management-observability
   - ../../base/objectbucket.io/objectbucketclaims/observability
+  - ../../base/objectbucket.io/objectbucketclaims/acm-metrics
```

Comment:
I'm thinking that we should move the `cluster-scope/overlays/nerc-ocp-infra/acm-metrics-storage` resources you created in the previous PR into `cluster-scope/bundles/acm-observability/acm-metrics-storage`, so that these things go together.

Reply:
I changed my mind, I think it's fine that we already configured the size of the storage in the infra cluster overlay.